scripts/README.md

```diff
@@ -3,3 +3,4 @@
 Common script utilities which can help automate common tasks within a BigQuery data warehouse.
 
 * [billing](/billing) - Example queries over the GCP billing export to help get you started using the standard data export format
+* [procedure](/procedure) - Example stored procedures that provide solution to address traditional database features
```

Comment:
Should we split this out into it's own top-level directory as outlined in [go/bigquery-utils](go/bigquery-utils)?

Reply:
Do you mean GoogleCloudPlatform/bigquery-utils/procedures?

Reply:
`GoogleCloudPlatform/bigquery-utils/stored_procedures`